commands/Moderation/mute.js

```diff
@@ -0,0 +1,36 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRole } = msg.guild.settings;
+  if (!msg.guild.settings.muteRole) return msg.reply("Sorry you have not created a mute role for this server.");
+  const value = user.roles.get(muteRole);
+  if (value && msg.guild.roles.get(muteRole)) {
```

Comment:
Would be better if you used Collection#has instead. However, if the member has the role, the guild itself has the role aswell. So this check is redundant.